driver-lambda/src/main/com/mongodb/lambdatest/LambdaTestApp.java

```diff
@@ -0,0 +1,143 @@
+    private double totalHeartbeatDuration = 0;
+    private double totalCommandCount = 0;
+    private double totalCommandDuration = 0;
+
+    public LambdaTestApp() {
+        String connectionString = System.getenv("MONGODB_URI");
+
+        MongoClientSettings settings = MongoClientSettings.builder()
+                .applyConnectionString(new ConnectionString(connectionString))
+                .addCommandListener(new CommandListener() {
+                    @Override
+                    public void commandSucceeded(@NonNull final CommandSucceededEvent event) {
+                        totalCommandCount++;
+                        totalCommandDuration += event.getElapsedTime(MILLISECONDS);
+                    }
+                    @Override
+                    public void commandFailed(@NonNull final CommandFailedEvent event) {
+                        totalCommandCount++;
+                        totalCommandDuration += event.getElapsedTime(MILLISECONDS);
+                    }
+                })
+                .applyToServerSettings(builder -> builder.addServerMonitorListener(new ServerMonitorListener() {
+                    @Override
+                    public void serverHeartbeatSucceeded(@NonNull final ServerHeartbeatSucceededEvent event) {
+                        totalHeartbeatCount++;
+                        totalHeartbeatDuration += event.getElapsedTime(MILLISECONDS);
+                    }
+                    @Override
+                    public void serverHeartbeatFailed(@NonNull final ServerHeartbeatFailedEvent event) {
+                        totalHeartbeatCount++;
+                        totalHeartbeatDuration += event.getElapsedTime(MILLISECONDS);
+                    }
+                }))
+                .applyToConnectionPoolSettings(builder -> builder.addConnectionPoolListener(new ConnectionPoolListener() {
+                    @Override
+                    public void connectionCreated(@NonNull final ConnectionCreatedEvent event) {
+                        openConnections++;
+                    }
+                    @Override
+                    public void connectionClosed(@NonNull final ConnectionClosedEvent event) {
+                        openConnections--;
+                    }
+                }))
+                .build();
+        mongoClient = MongoClients.create(settings);
+    }
+
+    public APIGatewayProxyResponseEvent handleRequest(final APIGatewayProxyRequestEvent input, final Context context) {
+        try {
+            MongoCollection<Document> collection = mongoClient
+                    .getDatabase("lambdaTest")
+                    .getCollection("test");
+            BsonValue id = collection.insertOne(new Document("n", 1)).getInsertedId();
+            collection.deleteOne(new Document("_id", id));
+
+            BsonDocument responseBody = new BsonDocument()
+                    .append("totalCommandDuration", new BsonDouble(totalCommandDuration))
```

Comment:
[nit] Naming this field as `totalCommandDurationMS` might help log readability, as it provides an indication of the measurement.